docs/antora/modules/ROOT/pages/management/source.adoc

```diff
@@ -16,6 +16,10 @@ For more details about PM2, see {url_pm2_docs}[PM2 docs].
 
 Moreover, we'll also talk about the steps to reset a Lisk Service node.
 
+TIP: The process ID of each PM2-managed process can be seen by executing the <<list>> command. To learn more about PM2 log management please refer the {url_pm2_management}[PM2 logs documentation].
+
+NOTE: To upgrade Lisk Service to a desired version, please check out the specific version with *`git checkout <tag>`*, and follow the existing process.
```

Comment:
What process are you referring to?

Reply:
please note that this page does not explain any specific process. it is just a collection of commands related to managing Lisk Service. So there is no particular process below.

Reply:
Yes, thats a good point. Then perhaps we could just amend the end of the note, to read as follows: 
"To upgrade Lisk Service to a desired version, please check out the specific version with *`git checkout <tag>`*, and then start Lisk Service as shown in the list of commands below.

Reply:
Please see my comment in the other PR: There should be at least a separate section explaining the process